client/webpack.client.rails.build.config.js

```diff
@@ -69,6 +69,9 @@ if (devBuild) {
   console.log('Webpack dev build for Rails'); // eslint-disable-line no-console
   config.devtool = 'eval-source-map';
 } else {
+  config.plugins.push(
+    new webpack.optimize.UglifyJsPlugin()
+  );
```

Comment:
@justin808 you're right, that was was unneeded -- I was misunderstanding the fact that the rails js helpers will serve different bundles depending on prod/dev environment. 

The good news is that running in production with just that single change to the rake task results in even greater savings:

before (www.reactrails.com)
<img width="870" alt="www" src="https://cloud.githubusercontent.com/assets/816444/13144647/d167685a-d619-11e5-9c11-6a074f6ed04e.png">

after
<img width="870" alt="www-after" src="https://cloud.githubusercontent.com/assets/816444/13144650/d624584e-d619-11e5-8047-770a35b767b6.png">

It would be good to have someone else verify that the reduction in file size is real -- lots of switching back and forth between branches, manually running commands, and taking screenshots is error-prone.

I'll remove the line for the webpack uglify plugin and squash those commits, if that sounds cool.